pkg/neorpc/result/peer_heights.go

```diff
@@ -0,0 +1,64 @@
+package result
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+type (
+	GetPeerHeights struct {
+		PeerHeights PeerHeights `json:"peerHeights"`
```

Comment:
peerheights or maybe just peers

Reply:
Or maybe even `connected`? An analogue of `getpeers` response, since here we return only connected peers info.